frontend/src/components/RotateCredentials.vue

```diff
@@ -0,0 +1,469 @@
+        return 'Unschedule Rotation'
+      }
+      if (this.maintenance) {
+        return 'Schedule Roatation'
+      }
+      switch (this.mode) {
+        case 'init':
+          return 'Initiate Roatation'
+        case 'complete':
+          return 'Complete Rotation'
+      }
+      return 'Rotate'
+    },
+    tooltip () {
+      if (this.isScheduledForMaintenance) {
+        return 'This operation is scheduled to be performed during the next maintenance time window'
+      }
+      if (this.isHibernationPreventingMaintenance && this.isShootStatusHibernated) {
+        return 'Cluster is hibernated. Wake up cluster to perform this operation.'
+      }
+      return undefined
+    },
+    componentTexts () {
+      const componentTexts = {
+        'rotate-kubeconfig-credentials': {
+          caption: this.isLoading ? 'Scheduling kubeconfig credentials rotation' : 'Start Kubeconfig Rotation',
+          errorMessage: 'Could not start the rotation of kubeconfig credentials',
+          successMessage: `Rotation of kubeconfig credentials started for ${this.shootName}`,
+          heading: 'Do you want to start the rotation of kubeconfig credentials?',
+          alert: {
+            type: 'warning',
+            messages: [
+              'The current kubeconfig credentials will be revoked.'
+            ]
+          }
+        },
+        'rotate-ca-start': {
+          caption: this.isLoading
+            ? 'Initiating certificate authorities rotation'
+            : this.isDisabled
+              ? 'Rotation already initiated'
+              : 'Initiate Certificate Authorities Rotation',
+          errorMessage: 'Could not initiate the rotation of certificate authorities',
+          successMessage: `Rotation of certificate authorities initiated for ${this.shootName}`,
+          heading: 'Do you want to initiate the rotation of certificate authorities?',
+          alert: {
+            type: 'info',
+            messages: [
+              'New CAs will be created and added to the bundle (together with the old CAs). Client certificates will be re-issued immediately.'
+            ]
+          }
+        },
+        'rotate-ca-complete': {
+          caption: this.isLoading
+            ? 'Completing certificate authorities rotation'
+            : this.isDisabled
+              ? 'Rotation initiation not completed'
+              : 'Complete Certificate Authorities Rotation',
+          errorMessage: 'Could not complete the rotation of certificate authorities',
+          successMessage: `Rotation of certificate authorities completed for ${this.shootName}`,
+          heading: 'Do you want to complete the rotation of certificate authorities?',
+          alert: {
+            type: 'warning',
+            messages: [
+              'Ensure that end-users have updated all cluster API clients that communicate with the control plane. The old CAs will be dropped from the bundle and server certificate will be re-issued.'
```

Comment:
```suggestion
              'Ensure that all parties (end-users, CD pipelines etc.) have updated their kubeconfig for this Shoot cluster since the CA rotation was last initiated. Otherwise the client requests will fail as the old CAs will be dropped from the bundle'
```

Reply:
Let's discuss this with @timebertt when we have this PR ready.
The texts are more or less copied from the gardner docu, but @rfranzke already indicated that there is room for improvement...

Reply:
done